src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/factory/events/ApplicationAppealMadeFactory.kt

```diff
@@ -0,0 +1,90 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.factory.events
+
+import io.github.bluegroundltd.kfactory.Factory
+import io.github.bluegroundltd.kfactory.Yielded
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.events.model.AppealDecision
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.events.model.AssessmentAppealed
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.events.model.PersonReference
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.randomDateTimeBefore
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.randomOf
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.randomStringMultiCaseWithNumbers
+import java.time.Instant
+import java.util.UUID
+
+class AssessmentAppealedFactory : Factory<AssessmentAppealed> {
+  private var applicationId: Yielded<UUID> = { UUID.randomUUID() }
+  private var applicationUrl: Yielded<String> = { randomStringMultiCaseWithNumbers(10) }
+  private var appealId: Yielded<UUID> = { UUID.randomUUID() }
+  private var appealUrl: Yielded<String> = { randomStringMultiCaseWithNumbers(10) }
+  private var personReference: Yielded<PersonReference> = { PersonReferenceFactory().produce() }
+  private var deliusEventNumber: Yielded<String> = { randomStringMultiCaseWithNumbers(6) }
+  private var createdAt: Yielded<Instant> = { Instant.now().randomDateTimeBefore(7) }
+  private var createdBy: Yielded<String> = { randomStringMultiCaseWithNumbers(10) }
```

Comment:
It might be more useful (and consistent e.g. with `application.submitted`) if we use the staff member here rather can a single string? e.g.

`private var createdBy: Yielded<StaffMember> = { StaffMemberFactory().produce() }`

Reply:
The change to the event structure to use `StaffMember` has been made.